homeassistant/components/demo/__init__.py

```diff
@@ -83,11 +83,11 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     if not hass.config.longitude:
         hass.config.longitude = 117.22743
 
-    tasks = [bootstrap.async_setup_component(hass, "sun", config)]
+    tasks = [setup.async_setup_component(hass, "sun", config)]
```

Comment:
`mypy` complains with this error when importing from `bootstrap`.
`Module has no attribute "async_setup_component"; maybe "async_setup_multi_components"?  [attr-defined]`

Since `bootstrap` is anyway importing it from `setup`, I have adjusted the code to import it directly from `setup`